src/lib/i18n.json

```diff
@@ -45,6 +45,9 @@
     "hld-reconcile-err-cmd-exe": "An error occurred while reconciling HLD.",
     "hld-reconcile-err-purge-repo-comps": "Could not purge hld repository component {0} in path {1}.",
 
+    "hld-install-manifest-pipeline-cmd-failed": "hld install-manifest-pipeline command was not successfully executed.",
+    "hld-install-manifest-pipeline-cmd-validate-github-repo-err": "Could not {0} or/and {1} were not supported.",
```

Comment:
Let's remove references to "github".

`"hld-install-manifest-pipeline-cmd-validate-repo-err": "Could not validate repositories: {0} and/or {1} may not  be supported."`

Reply:
accept